pyoverkiz/client.py

```diff
@@ -196,13 +205,9 @@ async def nexity_login(self) -> str:
         loop = asyncio.get_event_loop()
 
         # Request access token
-        client = await loop.run_in_executor(
-            None,
-            lambda: boto3.client(
-                "cognito-idp", config=Config(region_name=NEXITY_COGNITO_REGION)
-            ),
+        client = boto3.client(
```

Comment:
@tetienne why did this one get removed? This is currently breaking Nexity login.

Reply:
There was a typing error, and cannot found someone using our syntax, while this one yes :S

Reply:
I resolved it in a new PR :)